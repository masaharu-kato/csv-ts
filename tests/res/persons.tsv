id	name	age
1	John Tanaka	25
2	Taro Suzuki	31
3	Tom Sato	17
4	Judy Foobar	25
5	Unnamed	27
6	Sachi Piyo	20
