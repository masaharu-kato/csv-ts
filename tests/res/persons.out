John Tanaka, 25 years old, id is 1.
Taro Suzuki, 31 years old, id is 2.
Tom Sato, 17 years old, id is 3.
Judy Foobar, 25 years old, id is 4.
Unnamed, 27 years old, id is 5.
Sachi Piyo, 20 years old, id is 6.
